config.json

```
@@ -448,6 +448,17 @@
       "topics": null,
       "deprecated": true
     },
+    {
+      "slug": "linked-list",
+      "uuid": "840f4011-029a-4f5c-970f-f8e585cd4fdf",
+      "core": false,
+      "unlocked_by": "null",
+      "difficulty": 8,
+      "topics": [
+        "data_structures",
+        "lists",
+        "recursion"
```

This is a syntax error. Our CI should definitely have caught this.
```suggestion
        "recursion"
      ]
```